### DISCUSSION TITLE
Awkward floating icon for inspecting objects in Image panel

### DISCUSSION THREAD

**Discussion by defunctzombie** [foxglove/community#540] [Q&A]
The inspect object icon feels awkwardly placed. The existing zoom icon is a different distance from the bottom and side than this other icon.

It's also a different distance from the side as the other text overlay at the bottom. We should find some consistency in spacing with these UI elements.

<img width="510" alt="image" src="https://user-images.githubusercontent.com/84792/155053709-7fd296f5-469e-4520-bece-a286ac7990c4.png">

**Comment by foxymiles** [comment]
What should we do here? Maybe consolidate the zoom button and the inspect button into a common toolbar like the 3d panel?

**Comment by defunctzombie** [comment]
@2metres has some ideas around the image panel buttons and dropdowns. This should figure into those ideas.
